### PR TITLE
Javascript Classes and Styling

### PR DESCRIPTION
**Pull request by Othniel2471**
### In this pull request my partner and I:

- Refactored the index.js code and use javascript classes for the app functionality.
- Changed the html structure in the index.html and styled it to match the wireframe design provided.

### CODE REVIEW

**Review by julie-ify — COMMENTED** [review]
Hi @Othniel2471 and @marvin-nyalik,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### All the links in the headers of [the readme template](https://github.com/microverseinc/readme-template) should be present. You missing important header links and sections like Install, Usage, Future Features, etc. Kindly fix that. The loom walkthrough link on the header is broken. Kindly fix that by removing it or adding a proper loom walkthrough section.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by julie-ify — COMMENTED** [review]
Hi @Othniel2471 and @marvin-nyalik,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### The Future Features does not have a link on the headers. Kindly add it. Also remove the section description you have in that section i.e. "Describe 1 - 3 features you will add to the project."

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Othniel2471** [comment]
@julie-ify please can you specify what you mean about the "Describe 1 - 3 features you will add to the project.", am I meant to remove the texts I have in the yellow brackets?

**Comment by DammyShittu** [comment]
Hi @Othniel2471, yes you should remove the text: "Describe 1 - 3 features you will add to the project.". That's what the previous reviewer meant.

**Review by DammyShittu — COMMENTED** [review]
Hi @Othniel2471 and @marvin-nyalik,

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request

### You are required to remove the section description from the Future Features section as outlined by the previous reviewer.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: assets/js/index.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+  addBooks() {
+    const title = Books.bookTitle.value;
+    const author = Books.bookAuthor.value;
+    const bookDetails = new Books(title, author);
+
+    if (localStorage.getItem('books')) {
+      this.books = JSON.parse(localStorage.getItem('books'));
+    }
+    this.books.push(bookDetails);
```
Comment (zieeco) [inline assets/js/index.js]: - Hey team, while trying to add a book, this throws an error as it cannot read the undefined property
image
- Please, kindly check to fix this error.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Team,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- Nice using classes.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._




## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DammyShittu — APPROVED** [review]
Hi @Othniel2471 and @marvin-nyalik,

## APPROVED 🟢

## Highlights
✔️ Refactor to class

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**